pyproject.toml

```diff
@@ -46,7 +46,7 @@ dev = [
   "pytest-lazy-fixture",
   "strawberry-graphql[debug-server]==0.155.3",
   "pre-commit",
-  "arize",
+  "arize[AutoEmbeddings, LLM_Evaluation]",
```

Comment:
we might want to specify a version soon. I know we don't support pre-6 for example. Not blocking, just an idea